params.go

```diff
@@ -27,7 +27,7 @@ var (
 	)
 	initialCommonPool = flag.Float64(
 		"initialCommonPool",
-		100,
+		1000,
```

Comment:
why?

Reply:
(provide reason in desc)

Reply:
IIGO never ran ;( with 100

Reply:
![image](https://user-images.githubusercontent.com/33488131/104108525-3ff13100-52bd-11eb-99bc-75d2df4f7612.png)